e2e/data/temp2.txt

```diff
@@ -0,0 +1 @@
+Foobar
```

Comment:
Was this file accidentally added?

Reply:
No. This is just my file for uploading. If you think there is a better file to use, I'll happily swap it out.